inst/help/ConfirmatoryFactorAnalysis.md

```diff
@@ -3,15 +3,18 @@ Confirmatory Factor Analysis
 
 Confirmatory factor analysis (CFA) models observed variables (indicators) as noisy manifestations of underlying latent variables (factors). JASP's CFA is built on `lavaan` (lavaan.org; Rosseel, 2012), an `R` package for performing structural equation modeling. See Brown (2014) or Kline (2015) for books on the topic of CFA.
 
-Assignment Box
+### Assignment Box
```

Comment:
could you also reflect the changes in the dutch help files? you can use deepl

Reply:
I was afraid you were going to say that. Wasn't there the plan to have all the help files be translated the same way everything else is? I suppose that hasn't happened yet... :)